ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -2243,6 +2243,22 @@ keys:
                     transit:
                       type: bool
                       description: Enable EVPN multicast transit mode.
+            evpn_multicast_gateway_dr_election_algorithm:
+              type: dict
+              keys:
+                hrw:
```

Comment:
hrw is accepted by cli but i am not able to see the config in running-config. Can you please check

Reply:
@ClausHolbechArista the command in not reflecting in running config in our lab devices. Is it possible of some platform dependency?

Reply:
check with `show active all` in case it is default config